content/en/blog/2024/new-otel-features-envoy-istio/index.md

```diff
@@ -0,0 +1,246 @@
+---
+title: 'Observability on the Edge: Enhancing OTel support in Envoy and Istio'
+linkTitle: New OTel features in Envoy and Istio
```

Comment:
I like both the long and short titles, but they aren't mutually consistent and I think they should be. I'm not sure how to fix this though.

Reply:
Yeah looking now I agree with you. I will try to come up with something better.

Reply:
@chalin I tried a small change. Let me know what you think.

Reply:
Yes, better. I've suggested tweaks via https://github.com/open-telemetry/opentelemetry.io/pull/4553/files#r1627377010.